libs/code-demos/src/lib/multitab-editor/editor-from-model/editor-from-model.component.ts

```diff
@@ -71,9 +70,18 @@ export class EditorFromModelComponent
   }
 
   ngAfterViewInit() {
-    this.editor = this.editorInjector.editor = this.setUpEditor(
-      this.el.nativeElement
-    );
+    const options = {
+      ...MONACO_DEFAULTS,
+      model: this.model,
+      fontSize: this.fontSize
+    };
+
+    this.zone.runOutsideAngular(() => {
```

Comment:
How much of this can we reuse? Maybe move out to the service?

Reply:
@NothingEverHappens I will see if we can move instantiation call to a service.